src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -267,6 +267,36 @@ paths:
       operationId: create-prisoner-induction
       requestBody:
         $ref: '#/components/requestBodies/CreateInduction'
+    get:
+      summary: Return a Prisoner's Induction.
+      description: |
+        Returns a Prisoner's Induction.
+      tags:
+        - Inductions
+      responses:
+        '200':
+          $ref: '#/components/responses/Induction'
+        '404':
+          $ref: '#/components/responses/404ErrorResponse'
+      operationId: get-prisoner-induction
+
+  '/inductions':
```

Comment:
I'm not sure about this (the POST endpoint to get the induction summaries)
I wonder if we should whip it out of this PR and think about it separately. 

FWIW, my current thought process is that just because its in the current CIAG API, it doesn't mean we need to migrate it, and certainly not exactly as is. The only reason it's there is so that the Prisoner List screen - which is now _our_ Prisoner List screen in _our_ UI - knows if there is an induction. And it also calls the action plan summary endpoint for the same reason (to know if there are any Goals).
Those 2 summary endpoints were implemented specifically for the Prisoner List page so that it could get data from 2 sources to workout the status and routing.
It's no longer 2 sources .... so why do we need 2 API calls? 🤔 

I wonder if we'd be better off not implementing a copy of the summary API, and instead stepping back and thinking about what the UI needs and why, and designing a single API that addresses that ?
(in a separate story of course 👍 )

What do you think?

Reply:
That makes complete sense. I'll whip it out (as you put it!).